src/App.js

```diff
@@ -10,66 +10,66 @@ import Tile from "./components/Tile";
 import brand from './assets/brand.png'
 import our_story from './assets/our_story.png'
 function App() {
-  return (
-     <>
-         <h1>Handbags & Purses</h1>
-        <nav>
-         <Navigation
-             buttonName="to the collection"
-             disabled={false}/>
+    return (
```

Comment:
volgens mij is je styling toch wat meer naar links dan zoals het hoort?  b.v. onder <nav> zou <Navigation twee spaties naar links moeten etc